libs/comments/website/src/lib/comment-editor/comment-editor.tsx

```diff
@@ -109,100 +195,147 @@ export const CommentEditor = ({
+              }}
+            />
+
+            <Controller
+              name={'challenge.challengeSolution'}
+              control={control}
+              render={({field, fieldState: {error}}) => (
+                <TextField
+                  {...field}
+                  label={'Captcha'}
+                  error={!!error}
+                  helperText={error?.message}
+                />
+              )}
+            />
+          </CommentEditorChallengeWrapper>
         )}
-      />
-
-      {!hasUser && challenge?.data && (
-        <CommentEditorChallengeWrapper>
-          <CommentEditorChallenge
-            dangerouslySetInnerHTML={{
-              __html:
-                challenge.data.challenge.challenge
-                  ?.replace('#ffffff', 'transparent')
-                  .replace('width="200"', '')
-                  .replace('height="200"', '') ?? ''
+
+        {challenge?.error && <Alert severity="error">{challenge.error.message}</Alert>}
+
+        {error && <Alert severity="error">{error.message}</Alert>}
+
+        <CommentEditorActions>
+          <Button type="submit" size="small" endIcon={<MdSend />} disabled={loading}>
+            Kommentieren
+          </Button>
+
+          <Button
+            type="reset"
+            onClick={() => {
+              reset()
+              onCancel()
             }}
-          />
+            size="small"
+            variant="text"
+            color="secondary">
+            Abbrechen
+          </Button>
+        </CommentEditorActions>
+      </CommentEditorWrapper>
 
-          <Controller
-            name={'challenge.challengeSolution'}
-            control={control}
-            render={({field, fieldState: {error}}) => (
-              <TextField {...field} label={'Captcha'} error={!!error} helperText={error?.message} />
-            )}
-          />
-        </CommentEditorChallengeWrapper>
-      )}
-
-      {challenge?.error && <Alert severity="error">{challenge.error.message}</Alert>}
-
-      {error && <Alert severity="error">{error.message}</Alert>}
-
-      <CommentEditorActions>
-        <Button type="submit" size="small" endIcon={<MdSend />} disabled={loading}>
-          Kommentieren
-        </Button>
-
-        <Button
-          type="reset"
-          onClick={() => {
-            reset()
-            onCancel()
-          }}
-          size="small"
-          variant="text"
-          color="secondary">
-          Abbrechen
-        </Button>
-      </CommentEditorActions>
-    </CommentEditorWrapper>
+      <Modal open={modalOpen} onClose={handleClose}>
+        <ModalContent>
+          <CloseLogin onClick={handleClose}>
+            <MdClose />
+          </CloseLogin>
+          <LoginWrapper>
+            <LoginFormContainer afterLoginCallback={handleAfterLoginCallback} />
+            <Register>
+              <Button css={buttonStyles} onClick={registerRedirect}>
+                <IconButton css={iconStyles} size="large">
```

Comment:
Button is nested in button. Thats not valid HTML.

You can simply use Button and give it a start icon.

Reply:
<img width="656" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/eb44d8e2-b899-4697-b09b-91e9539bf3c1">